docs/markdown/Wrap-dependency-system-manual.md

```diff
@@ -147,6 +149,37 @@ wrap-file mode. When using wrap-git, the repository must contain all
 Meson build definitions. Since *0.55.0* Meson build patches are
 supported for any wrap modes, including wrap-git.
 
+## Diff files
+
+*Since: 0.61.0*
+
+You can also provide local patch files in `diff` format. For historic reasons,
+they are referred to as "diff files", since the "patch" name is already used for
+Meson build patches.
```

Comment:
Due to precisely this confusion I prefer to avoid the name "build patch" and use "overlay archive".

Reply:
Changed to "overlay archives".